src/main/java/it/cnr/isti/hpc/wikipedia/article/Link.java

```diff
@@ -50,19 +53,18 @@ public void setId(String id) {
 	}
 	
 	
-	public String getDescription() {
-		return this.description;
+	public String getAnchor() {
+		return this.anchor;
 	}
 	
 	
-	public void setDescription(String description) {
+	public void setAnchor(String anchor) {
         // Some links do not have any anchor
         // For those cases the anchor is the same wikipedia Id
-		if (description.isEmpty()){
-            this.description = this.id.replace("_", " ");
-        }else{
-            this.description = description;
-        }
+        if (anchor == null || anchor.isEmpty())
+           this.anchor = this.id.replace("_", " ");
```

Comment:
Is this the mechanism for handling empty links ? If so, I wonder whether this would not generate false links for a system that presuppose that the entities linked would appear in wikipedia.